ambry-account/src/test/java/com/github/ambry/account/HelixAccountServiceTest.java

```diff
@@ -19,6 +19,7 @@
 import com.github.ambry.config.HelixPropertyStoreConfig;
 import com.github.ambry.config.VerifiableProperties;
 import com.github.ambry.utils.Utils;
+import com.google.common.collect.Sets;
```

Comment:
Looks like it isn't - the line that uses this can be replaced with standard java library code I think